android/src/main/java/app/shosetsu/android/ui/reader/ChapterReader.kt

```diff
@@ -249,13 +250,14 @@ class ChapterReader
 								//item { viewModel.horizontalSwitchOption() }
 								item { viewModel.invertChapterSwipeOption() }
 								item { viewModel.readerKeepScreenOnOption() }
+								item { viewModel.enableFullscreen() }
 								item { viewModel.showReaderDivider() }
 								item { viewModel.stringAsHtmlOption() }
 								item { viewModel.doubleTapFocus() }
-								item { viewModel.doubleTapSystem() }
+								item { viewModel.doubleTapSystem(enableFullscren) }
```

Comment:
The premise of the viewnodel extension here is removal of redundant boiler plate for observing the view model. Move observation into the setting item itself.

Reply:
👍